docs/src/development/file-transfer.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Transfer files to and from your app"
+weight: 7
+sidebarTitle: Transfer files
+---
+
+After your app is built, its file system is read-only.
+This means that the only way you can edit your app's files and code is through Git.
+
+However, you can transfer files to and from your built app without using Git.
+
+To do so, you need to configure [mounts](../create-apps/app-reference.md#mounts).
```

Comment:
:hammer_and_wrench:  This seems to imply that both to and from require mounts because the `so` here seems to cover both. I think this change might solve that? But could be something better :shrug: 
```suggestion
To transfer files to the app, you need to configure [mounts](../create-apps/app-reference.md#mounts).
```

Reply:
See new introduction rework for improved clarity.